src/Agent.Sdk/ProcessInvoker.cs

```diff
@@ -805,6 +810,54 @@ private void NixKillProcessTree()
             }
         }
 
+        private void WriteProcessOomScoreAdj(Process proc)
+        {
+            try
+            {
+                if (proc.StartInfo.FileName.Contains("Agent.Worker", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
have you consider add a parameter to the process invoker like `bool highPriority=false`, so the consumer of process invoker can decide this?

Reply:
Switched to this pattern